pkg/apihelpers/apihelpers.go

```diff
@@ -8,8 +8,73 @@ import (
 	"fmt"
 
 	mcfgv1 "github.com/openshift/api/machineconfiguration/v1"
+	opv1 "github.com/openshift/api/operator/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/klog/v2"
+)
+
+var (
+	// This is the list of MCO's default node disruption policies.
+	defaultClusterPolicies = opv1.NodeDisruptionPolicyClusterStatus{
+		Files: []opv1.NodeDisruptionPolicyStatusFile{
+			{
+				Path: "/etc/mco/internal-registry-pull-secret.json",
+				Actions: []opv1.NodeDisruptionPolicyStatusAction{
+					{
+						Type: opv1.NoneStatusAction,
+					},
+				},
+			},
+			{
+				Path: "/var/lib/kubelet/config.json",
```

Comment:
minor: given that these paths are referenced in a lot of places, maybe it's worth it to make them an operator-level constant.

Although maybe eventually we'd only want less references to these paths?

Reply:
Will see if I can tighten these up!